language/tools/move-mv-llvm-compiler/src/stackless/translate.rs

```diff
@@ -458,12 +453,9 @@ impl<'mm, 'up> ModuleContext<'mm, 'up> {
                 // but might end up broken in the future.
                 self.llvm_cx.int8_type()
             }
```

Comment:
I believe the below change is incorrect in the case of external structs-- those structs referenced in the current module but defined in a different module.

Reply:
Oops, I did not realize if was providing support for the externally defined structs. Will bring it back.

Reply:
Yes-- external structs are actually the common case in Move. Most modules will import some sort of standard type provided by a particular chain, say `my_favorite_chain_library::Coin` or `foo::LiquidityPool`.